### PR TITLE
TASK: Allow to configure if processed images must be interlaced

### PR DESCRIPTION
**Pull request by dfeyer**
This change adds a new setting in the Media package to enable image interlacing.

It's disabled by default, but you can change the setting to one of the values in
`Neos.Media.image.defaultOptions.interlace`:

- `%\Imagine\Image\ImageInterface::INTERLACE_NONE%` (default)
- `%\Imagine\Image\ImageInterface::INTERLACE_LINE%`
- `%\Imagine\Image\ImageInterface::INTERLACE_PLANE%`
- `%\Imagine\Image\ImageInterface::INTERLACE_PARTITION%`

### CODE REVIEW

File: Neos.Media/Configuration/Settings.yaml [hunk 1]
```diff
@@ -72,6 +72,7 @@ Neos:
         # Image quality, from 0 to 100
         quality: 90
         convertCMYKToRGB: true
+        interlace: '%\Imagine\Image\ImageInterface::INTERLACE_NONE%'
```
Comment (mficzel) [inline Neos.Media/Configuration/Settings.yaml]: Not sure about that i like using a constand of a library here. Why not just add "none" explain the options as with quality above.

**Comment by mficzel** [comment]
Would be cool to document this to the documentation of the mediaPackage aswell.

File: Neos.Media/Configuration/Settings.yaml (hunk 1 above)
Comment (dfeyer) [reply Neos.Media/Configuration/Settings.yaml]: Consistant with the Filter configuration

File: Neos.Media/Documentation/ConfigureImageGeneration.rst [hunk 2]
```diff
@@ -29,3 +29,35 @@ If you are working with CMYK images and don't like to convert them automatically
...
+
+
+Changed default filter for Image processing
+===========================================
+
+If you have configured a Imagine driver that support alternative filter (this the case is you use Imagick or Gmagick), you can select the filter within your Settings.yaml:
+
+.. code-block:: yaml
+
+    TYPO3:
```
Comment (kdambekalns) [inline Neos.Media/Documentation/ConfigureImageGeneration.rst]: Neos, no?

File: Neos.Media/Documentation/ConfigureImageGeneration.rst [hunk 3]
```diff
@@ -29,3 +29,35 @@ If you are working with CMYK images and don't like to convert them automatically
...
+more informations about each filter. Check the \Imagine\Image\ImageInterface to know with filter are supported by Imagine.
+
+Produce interlaced images
+=========================
+
+To generate progressive images you can configure the driver within your Settings.yaml:
+
+.. code-block:: yaml
+
+    TYPO3:
```
Comment (kdambekalns) [inline Neos.Media/Documentation/ConfigureImageGeneration.rst]: Neos!

File: Neos.Media/Configuration/Settings.yaml (hunk 1 above)
Comment (mficzel) [reply Neos.Media/Configuration/Settings.yaml]: You are right

**Comment by msauerlaudert** [comment]
On our new website (build with Neos 3.2.1) we needed interlaced images. I've implemented it in the system and it's works like a charm. Need this in the next Neos release!

**Comment by dfeyer** [comment]
@mficzel @kdambekalns Last review and merge please ;) thanks for the review

**Pull request merged**